chore(NA): update kbn plugin helpers readme file (#83083)
elastic/kibana: packages/kbn-plugin-helpers/README.md

## Installation

> To install the plugin helpers just create the needed npm scripts on your plugin's `package.json` (as exemplified below) which 

Reviewer: ```suggestion
You don't actually need to install the plugin helpers, they are automatically inherited from the Kibana project by building your plugin within the Kibana repo. To use the plugin helpers just create the needed npm scripts on your plugin's `package.json` (as exemplified below) which 
```

Author: 👍

is already the case if you use the new `node scripts/generate_plugin` script.

```json
{
  "scripts" : {
    "build": "yarn plugin-helpers build",
    "plugin-helpers": "node ../../scripts/plugin_helpers",
    "kbn": "node ../../scripts/kbn"
  }
}
```

> This will make it possible to use the package from the repository into your plugin, but the `plugin-helpers` executable won't be available until you make sure you have run bootstrap at least once.

Reviewer: ```suggestion
This will make it easier to execute the `plugin-helpers` script from within your plugin repository.
```
